process-document/src/main/kotlin/com/ritense/processdocument/service/ProcessDocumentsService.kt

```diff
@@ -30,6 +31,10 @@ class ProcessDocumentsService(
     private val camundaProcessService: CamundaProcessService,
     private val associationService: ProcessDocumentAssociationService
 ) {
+    fun startProcessByProcessDefinitionKeyWithExecution(processDefinitionKey: String, execution: DelegateExecution) {
```

Comment:
I think if there's variables, they should just be provided, This can be seen on like 44, so I think this method should be removed.